examples/react-externals-web/src/App.css

```
@@ -0,0 +1,119 @@
+  width: 100px;
+  height: 100px;
+  animation: Logo--spin infinite 20s linear;
+}
+
+.Logo--lynx {
+  width: 100px;
+  height: 100px;
+  animation: Logo--shake infinite 0.5s ease;
+}
+
+@keyframes Logo--spin {
```

Stylelint (17.14.0)

[error] Expected keyframe name "Logo--spin" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)